calcom/cal.com
# fix: Better UI for the setup page, When trying to reach an app that hasn't been already installed under an account

## What does this PR do?

I have updated the setup page for the /apps/make/setup as per the figma design

- FIxes #15973 

## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?



- Are there environment variables that should be set?
- No
- What are the minimal test data to have?
- NA
- What is expected (happy path) to have (input and output)?
- On the route - /apps/make/setup
- Any other important info that could help to test that PR
- No

## Checklist
At first the page used to look like this - 

![image](https://github.com/user-attachments/assets/73275f93-1e69-41d1-a6b0-54c4e2aed82f)

and now its looking as per the figma design - 

![image](https://github.com/user-attachments/assets/a633f9a1-d954-4c75-835e-21da43bfb62d)
